onnxscript/type_annotation.py

```diff
@@ -146,3 +147,52 @@ def get_return_types(typeinfo: type | typing.Sequence[type]) -> typing.Sequence[
     if get_origin(typeinfo) is tuple:
         return get_args(typeinfo)
     return (typeinfo,)
+
+
+def _reduce_type_var_to_union(hint: typing.TypeVar):
+    """Reduce a TypeVar to a Union type on which we can use issubclass to check membership."""
```

Comment:
change "Union type" to "type" or "type bound" ? It doesn't always seem to be a "Union" below.

Reply:
Removed the function instead